### DISCUSSION TITLE
Not a valid option duration of the flight

### DISCUSSION THREAD

**Discussion by praneeth300** [ketangangal/FlightFareEstimatorProject#7] [Ideas]
Hey I seen your entire project and heroku link, but I found you added a option "duration" this is not a valid option because we can't say that how much time it would be take through. Also only few cities are there, it is better to add more cities, my small suggestion I hope it will help you to get better insights

**Comment by ketangangal** [comment]
Hey Praneeth, 
We had limited data from ineuron so based on that we had created this project. If in future we get api from airline for sure we will add more fields. In duration field we are asking for duration as cost always depends on time! 
Hope it helps.
